### PR TITLE
make the ceph-deploy job upload to chacra

### PR DESCRIPTION
**Pull request by andrewschoen**

### CODE REVIEW

File: ceph-deploy/build/build [hunk 1]
```diff
@@ -38,17 +41,26 @@ then
...
+            exit 0
+        fi
 
         if [ ! -e setup.py ] ; then
             echo "Are we in the right directory"
             exit 1
         fi
 
-        if ! CREATEREPO=`which createrepo` ; then
-            echo "Please install the createrepo package"
```
Comment (alfredodeza) [inline ceph-deploy/build/build]: lol dependency management

File: ceph-deploy/build/build [hunk 2]
```diff
@@ -38,17 +41,26 @@ then
...
-            exit 4
```
Comment (alfredodeza) [inline ceph-deploy/build/build]: magic number alert

Comment (ktdreyer) [inline ceph-deploy/build/build]: goodbye 4, we will not miss you

File: ceph-deploy/build/build [hunk 3]
```diff
@@ -120,40 +137,11 @@ then
...
-    # Build Repo
-    PKG=../ceph-deploy*.changes
-    mkdir -p $REPO/conf
-    if [ -e $REPO/conf/distributions ] ; then
-        rm -f $REPO/conf/distributions
-    fi
+    [ "$FORCE" = true ] && chacra_flags="--force" || chacra_flags=""
+
+    # push binaries to chacra
+    find ../ | egrep "*\.(changes|deb|dsc|gz)$" | egrep -v "(Packages|Sources|Contents)" | chacractl binary ${chacra_flags} create ${chacra_endpoint}
```
Comment (alfredodeza) [inline ceph-deploy/build/build]: could this be a path determined from `$WORKSPACE` vs. using something relative like `../` ? I know I've spent a lot of times trying to figure out the exact location before and normalizing in using `$WORKSPACE`

File: ceph-deploy/build/build [hunk 4]
```diff
@@ -120,40 +137,11 @@ then
...
 
-    # FIXME: This file is explicitly saying what it was signed with but we no longer
```
Comment (alfredodeza) [inline ceph-deploy/build/build]: zomg thank you

**Pull request merged**